### PR TITLE
Fix lint

### PR DESCRIPTION
**Pull request by lucasgsfelix**
Some errors I couldn't solve, and I two of the errors presents are caused by isort and pylint conflict:

setup.py|104:0|Too few public methods (1/2) (R0903, too-few-public-methods) [pylint]
	stats.py|None:None|Imports are incorrectly sorted. [isort] ### the conflict that I was talinkg about
	stats.py|10:0|No name 'MultipartType' in module 'pyof.v0x01.controller2switch.common' (E0611, no-name-in-module) [pylint]
	stats.py|10:0|No name 'v0x01' in module 'pyof.v0x01.controller2switch.common' (E0611, no-name-in-module) [pylint]
	stats.py|15:0|No name 'v0x04' in module 'pyof.v0x04.controller2switch' (E0611, no-name-in-module) [pylint]
	stats.py|274:15|MultipartRequest is not callable (E1102, not-callable) [pylint]
	stats.py|279:4|Parameters differ from overridden 'listen' method (W0221, arguments-differ) [pylint]
	stats.py|324:4|Parameters differ from overridden 'listen' method (W0221, arguments-differ) [pylint]
	stats.py|358:15|MultipartRequest is not callable (E1102, not-callable) [pylint]
	stats.py|363:4|Parameters differ from overridden 'listen' method (W0221, arguments-differ) [pylint]
	stats_api.py|61:52|Instance of 'StatsAPI' has no '_dpid' member (E1101, no-member) [pylint]
	stats_api.py|63:61|Instance of 'StatsAPI' has no '_dpid' member (E1101, no-member) [pylint]
	stats_api.py|78:27|Using possibly undefined loop variable 'col' (W0631, undefined-loop-variable) [pylint]
	stats_api.py|164:4|Parameters differ from overridden '_get_latest_stats' method (W0221, arguments-differ) [pylint]
	stats_api.py|262:4|Parameters differ from overridden '_get_latest_stats' method (W0221, arguments-differ) [pylint]
	tests/test_rrd.py|5:0|Imports from package unittest are not grouped (C0412, ungrouped-imports) [pylint]
	tests/test_user_speed.py|5:0|Imports from package unittest are not grouped (C0412, ungrouped-imports) [pylint] isort conflict
 user_speed.py|7:0|Too few public methods (1/2) (R0903, too-few-public-methods) [pylint]

### CODE REVIEW

File: stats_api.py [hunk 1]
```diff
@@ -195,8 +196,9 @@ def _get_speed(self, iface):
             iface.set_custom_speed(user_speed)
         return iface.speed
 
-    def _add_utilization(self, row, iface):
+    def _add_utilization(self, row):
```
Comment (hdiogenes) [inline stats_api.py:199]: Please make sure that no other method calls `_add_utilization()` with the `iface` parameter.

File: tests/test_rrd.py [hunk 2]
```diff
@@ -33,10 +33,10 @@ def test_2_points_out_of_3(self, get_rrd_method):
 
         def update_rrd(multiplier):
             """Update rrd."""
-            rx = tx = multiplier * STATS_INTERVAL
-            tstamp = start + rx
+            rx_arg = tx_arg = multiplier * STATS_INTERVAL
+            tstamp = start + rx_arg
             # any value for index is OK
-            rrd.update([None], tstamp, rx=rx, tx=tx)
+            rrd.update([None], tstamp, rx=rx_arg, tx=tx_arg)
```
Comment (hdiogenes) [inline tests/test_rrd.py]: RX and TX are not really arguments to this method. Maybe `rx_val` or `rx_value` are better names.

**Review by hdiogenes — CHANGES_REQUESTED** [review]

File: tests/test_rrd.py (hunk 2 above)
Comment (lucasgsfelix) [reply tests/test_rrd.py]: I updated the files.

File: stats_api.py (hunk 1 above)
Comment (lucasgsfelix) [reply stats_api.py:199]: I checked and it's ok.

**Comment by hdiogenes** [comment]
@lucasgsfelix, the tests are failing with `Could not open requirements file: [Errno 2] No such file or directory: 'requirements.txt'`

**Comment by lucasgsfelix** [comment]
> @lucasgsfelix, the tests are failing with `Could not open requirements file: [Errno 2] No such file or directory: 'requirements.txt'`

This error was happening before I make some changes in the napp. I copied 'requeriments/dev.txt' to the source folder and changed the name and this error was fixed, however I didn't know for sure if this was the best way to fix the problem. After I fix this problem I found another problem while running scruntinizer in the napp. The problem was a module import error.

**Comment by hdiogenes** [comment]
@lucasgsfelix, please remove `.scrutinizer.yml`. This file has custom Scrutinizer settings that are outdated and cause errors.

**Comment by lucasgsfelix** [comment]
> @lucasgsfelix, please remove `.scrutinizer.yml`. This file has custom Scrutinizer settings that are outdated and cause errors.

I removed the file.

**Pull request merged**